indra/llui/lltextbase.cpp

```diff
@@ -2349,10 +2353,9 @@ void LLTextBase::appendLineBreakSegment(const LLStyle::Params& style_params)
 
 void LLTextBase::appendImageSegment(const LLStyle::Params& style_params)
 {
-	if(getPlainText())
```

Comment:
Outr coding standard requests "Use braces for clarity, even when not strictly required", removing braces here goes against that.